drivers/i2s/i2s_ll_stm32.c

```diff
@@ -950,7 +956,8 @@ static const struct i2s_stm32_cfg i2s_stm32_config_##index = {		\
 	.pclk_len = DT_INST_NUM_CLOCKS(index),				\
 	.pcfg = PINCTRL_DT_INST_DEV_CONFIG_GET(index),			\
 	.irq_config = i2s_stm32_irq_config_func_##index,		\
-	.master_clk_sel = DT_INST_PROP(index, mck_enabled)		\
+	.master_clk_sel = DT_INST_PROP(index, mck_enabled),		\
+	.ioswp = DT_INST_PROP(index, ioswp)				\
```

Comment:
Could you add a terminal coma. It will prevent next addition in this structure to modify your line (as you needed to with the previous line).